cmd/root.go

```diff
@@ -266,3 +269,15 @@ func readFromFile(filePath string) (string, error) {
 	}
 	return strings.TrimSpace(string(content)), nil
 }
+
+func formatSwaggerError(e error) error {
+	obj, err := reflections.GetField(e, "Payload")
```

Comment:
can you please try to do the same reflection lookup using the reflect package from stdlib ?

Reply:
Native code is here:
https://github.com/oleiade/reflections/blob/master/reflections.go#L24

Reply:
I actually was quite happy to find this library because I think reflection in Go looks complex even for the simplest of things. This library just does the right thing, I guess, and has no other dependencies.^^

Reply:
sure, but this library is unmaintained since 2016, i really want to omit it.

Reply:
another idea:
as we need this mapping in other consumers of the metal-api as well, wouldnt it make sense to provide a helper function in metal-go to format this error message ?

Reply:
We can do this but we always should try to keep the original error type in order to be able to do type switches on "not found" and so on. This is really just a generic attempt to improve printing without having a consumer check all possible error types of our API.

Reply:
OK